tools/WebJobs.Script.Performance/WebJobs.Script.Performance.Dashboard/ReportProcessorcs.cs

```diff
@@ -0,0 +1,101 @@
+﻿using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Blob;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace WebJobs.Script.Tests.Perf.Dashboard
+{
+    public class ReportProcessor
+    {
+        private static Dictionary<string, double> RPSThreshold = new Dictionary<string, double>()
+        {
+            { "C# Ping (VS)", 130},
```

Comment:
These functions are defined externally, right? Would be nice to avoid the coupling here. Ideally this would be added to the report or some other source we load here.